expertise/create_dataset.py

```diff
@@ -300,12 +305,55 @@ def retrieve_expertise(self):
 
         return expertise
 
+    def get_papers_from_group(self, submission_group):
+        client = self.openreview_client
+        group_a_members = client.get_group(submission_group).members
+        pbar = tqdm(total=len(group_a_members), desc=submission_group + ' status...')
+        publications_by_profile_id = {}
+        all_papers = []
+
+        def get_status(profile_id):
+            pbar.update(1)
+            profile = openreview.tools.get_profile(client, profile_id)
+            if profile:
+                publications = list(openreview.tools.iterget_notes(self.openreview_client, content={'authorids': profile.id}))
+                return { 'profile_id': profile_id, 'papers': publications }
+        futures = []
+        with ThreadPoolExecutor(max_workers=self.config.get('max_workers')) as executor:
+            for profile_id in group_a_members:
+                futures.append(executor.submit(get_status, profile_id))
+        pbar.close()
+
+        for future in futures:
+            result = future.result()
+            # Convert publications to json
+            papers_json_list = []
+            for paper_note in result['papers']:
+                papers_json_list.append(paper_note.to_json())
+            publications_by_profile_id[result['profile_id']] = papers_json_list
+            all_papers = all_papers + result['papers']
+
+        # Dump publications by profile id
+        with open(self.root.joinpath('publications_by_profile_id.json'), 'w') as f:
+            json.dump(publications_by_profile_id, f, indent=2)
+        
+        return all_papers
 
     def get_submissions(self):
         invitation_ids = self.convert_to_list(self.config.get('paper_invitation', []))
         paper_id = self.config.get('paper_id')
+        submission_group = None
         submissions = []
 
+        # If there is more than 1 group, set the second group as the submission group
+        group_ids = self.convert_to_list(self.config.get('match_group', []))
```

Comment:
isn't `match_group` being used when we compute scores between reviewers and submissions? 

is this going to load all the papers for all the reviewers?

Reply:
A reviewer-submission scoring job would have fields `match_group: ['ABC.cc'], paper_invitation: 'ABC.cc/-/Submission'` whereas a group-group scoring job would have no paper_invitation and only have `['ABC.cc/Senior_Area_Chairs', 'ABC.cc/Area_Chairs']`

So when there are at least 2 groups present in the `match_group` array, the way this is adapted into the reviewer-submission framework is that the first group acts as the reviewers and all publications of all members in the second group acts as the submissions